resources/api-gateway/files/crd-apirules.yaml

```diff
@@ -433,7 +433,7 @@ spec:
                     type: array
                   path:
                     description: Path to be exposed
-                    pattern: ^/([0-9a-zA-Z./*]+)
+                    pattern: ^([0-9a-zA-Z./*(]+)
```

Comment:
Why is the opening parenthesis `(` in the list?

Reply:
The parenthesis is on the list because it is required for negative lookahead. For example:
```
- path: (?!.*/suffix.ico\s)/.*
```

Reply:
OK, but then we also need some other characters in the character class: `)`, `?`, `!`, `\`
Perhaps the pattern should be then: `^([0-9a-zA-Z./*()?!\\_-]+)` ?